rhel/SPECS/kubectl.spec

```diff
@@ -0,0 +1,127 @@
+%global with_debug   0
+
+%if 0%{?with_debug}
+# https://bugzilla.redhat.com/show_bug.cgi?id=995136#c12
+%global _dwz_low_mem_die_limit 0
+%else
+%global debug_package %{nil}
+%endif
+
+%global provider                github
+%global provider_tld            com
+%global project                 kubernetes
+%global repo                    kubernetes
+# https://github.com/kubernetes/kubernetes
+
+%global provider_prefix         %{provider}.%{provider_tld}/%{project}/%{repo}
+%global import_path             k8s.io/kubernetes
+%global commit                  ec6eb119b81be488b030e849b9e64fda4caaf33c
+%global shortcommit              %(c=%{commit}; echo ${c:0:7})
+
+%global install_golang 0
+
+# Needed otherwise "version_ldflags=$(kube::version_ldflags)" doesn't work
+%global _buildshell  /bin/bash
+%global _checkshell  /bin/bash
+
+
+Name:           kubectl
+Version:        1.16.8
+Release:        1%{?dist}
+Summary:        CLI client for Kubernetes
+License:        ASL 2.0
+URL:            https://github.com/kubernetes/kubernetes
+ExclusiveArch:  x86_64 %{arm}
+Source0:        https://github.com/kubernetes/kubernetes/archive/%{commit}/%{repo}-%{shortcommit}.tar.gz
+
+%if %{install_golang}
+BuildRequires:   golang >= 1.12.0
+%endif
+BuildRequires: go-bindata
```

Comment:
go-bindata should be installed into GOPATH and better to be done via go get
and dependency should be moved to install_golang if statement

Reply:
go-bindata is installed from EPEL, because build scripts prefere this